src/microbe_stage/Microbe.cs

```diff
@@ -198,28 +265,13 @@ public class Microbe : RigidBody, ISpawned, IProcessable, IMicrobeAI, ISaveLoade
     public float MovementFactor { get; private set; } = 1.0f;
 
     /// <summary>
-    ///   If true cell is in engulf mode.
+    ///   The current mode of the microbe. Shared across the colony
     /// </summary>
-    /// <remarks>
-    ///   <para>
-    ///     Prefer setting this instead of directly setting the private variable.
-    ///   </para>
-    /// </remarks>
     [JsonIgnore]
-    public bool EngulfMode
+    public MicrobeAction MicrobeMode
```

Comment:
I think this should be renamed to `State` and the action to `MicrobeState`, because Unbinding is not an "action" really. And having a "State" is probably familiar to many developers.